Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -161,6 +161,8 @@ class InitialValues {
     static CSS::TableLayout table_layout() { return CSS::TableLayout::Auto; }
     static QuotesData quotes() { return QuotesData { .type = QuotesData::Type::Auto }; }
     static CSS::TransformBox transform_box() { return CSS::TransformBox::ViewBox; }
+    static CSS::Size x() { return CSS::Size::make_px(0); }
+    static CSS::Size y() { return CSS::Size::make_px(0); }
```

Comment:
Thinking about it, these should be `LengthPercentage` instead of `Size`, right? Same for the other occurrences in this file. It's a position not a size. (And it's now hitting me how the name "length" is wrong, but that's what the spec calls it so eh.)

Reply:
SVG is a little odd with units, they do sometimes use `<number>|<percentage>` too (but here units like `px` also appear to be allowed).

Reply:
Yeah, there's a place in the spec somewhere that explains how to parse things differently when in an svg context. I thought I might have implemented that but maybe I only thought about it. The unitless-length quirk works instead for now.